shinkai-libs/shinkai-sqlite/src/lib.rs

```diff
@@ -211,6 +211,7 @@ impl SqliteManager {
         Self::migrate_tools_table(conn)?;
         Self::migrate_agents_table(conn)?;
         Self::migrate_llm_providers_table(conn)?;
+        Self::migrate_mcp_servers_table(conn)?;
```

Comment:
is this always going to migrate no matter what? maybe we need an if statement ot check if it is actually necessary -- we may have already upgraded ut

Reply:
oohh sorry I though we were in some way storing the migrations and doing something more advanced. Now I added a a trick to check if HTTP type was previously added